.github/workflows/pypi.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v1
         with:
-          python-version: '3.9'
+          python-version: "3.9"
```

Comment:
We are always using `'` instead of `"`, and we should not use any `'` or `"` if we can omit it.

Does prettier support the above rule?